packages/core/src/loader/loadMocks.ts

```diff
@@ -0,0 +1,39 @@
+import { readdir } from "fs-extra";
+import { resolve } from "node:path";
+import { cwd } from "node:process";
+import * as tsImport from 'ts-import';
+import { server } from "../index";
+
+export const start = async () => {
+  try {
+    await server.listen({ port: 3100 });
+  } catch (err) {
+    server.log.error(err);
+    process.exit(1);
+  }
+};
+
+async function* getFiles(dir: string): any {
```

Comment:
You might want to use glob instead.
https://www.npmjs.com/package/glob

Reply:
Nice thanks. Doing it now.